### PR TITLE
Dockerize the bot.

### PR DESCRIPTION
**Pull request by 57-Wolve**
Dockerising the bot.

### CODE REVIEW

**Comment by Senither** [comment]
👋 Cheers for the PR and sorry for the slow response.

When running the bot using the Docker file, it seems to create the `config.yml` volume as a directory instead of the file, this is most likely due to the configuration file not being copied out before the volume is created.

There are already plans for adding Docker support in Ava, however it would most likely have to be done through [Docker Composer](https://docs.docker.com/compose/), as that would allow starting up a Database and Lavalink container at the same time, which would help produce a more real production environment, and overall give a better experience to users.

I'll leave the PR open for if you wanna fix the `config.yml` volume, otherwise feel free to close the PR.

**Comment by sweepyoface** [comment]
> There are already plans for adding Docker support in Ava, however it would most likely have to be done through [Docker Composer](https://docs.docker.com/compose/), as that would allow starting up a Database and Lavalink container at the same time, which would help produce a more real production environment, and overall give a better experience to users.

You would tag an image for the bot itself and provide an *example* Compose file, thus this PR isn't blocked by that. Just wanted to point that out.

**Comment by 57-Wolve** [comment]
> 
> 
> wave Cheers for the PR and sorry for the slow response.
> 
> When running the bot using the Docker file, it seems to create the `config.yml` volume as a directory instead of the file, this is most likely due to the configuration file not being copied out before the volume is created.
> 
> There are already plans for adding Docker support in Ava, however it would most likely have to be done through [Docker Composer](https://docs.docker.com/compose/), as that would allow starting up a Database and Lavalink container at the same time, which would help produce a more real production environment, and overall give a better experience to users.
> 
> I'll leave the PR open for if you wanna fix the `config.yml` volume, otherwise feel free to close the PR.

How am I just now seeing this. 0_o

I noticed that issue with the config.yml but was to busy at the time to fix it, I'll add theses plans to my Jira and work on them. :)

File: Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ RUN chmod a+x buildpack-run.sh; \
 	./buildpack-run.sh; \
 	gradle build --stacktrace
 
-FROM anapsix/alpine-java AS runtime
```
Comment (57-Wolve) [inline Dockerfile]: Removed because this docker image is deprecated.

**Comment by 57-Wolve** [comment]
TODO:
- Save config.yml in volume for persistence and easy editing.
- Create Docker Compose YML file and configurations.

**Pull request closed without merging**